modules/installation-configure-proxy.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_aws/installing-aws-customizations.adoc
+// * installing/installing_aws_user_infra/installing-aws-user-infra.adoc
+// * installing/installing_bare_metal/installing-bare-metal.adoc
+// * installing/installing_vsphere/installing-vsphere.adoc
+// TODO: gcp
+
+// Questions:
+// - Can they provide multiple certs in additionalTrustBundle?
+// - Is it true that the trustedCA field is filled in? Slides say "trusted-ca-bundle",  but ConfigMap created is user-ca-bundle
```

Comment:
Yes, a proxy is created with `trustedCA` referencing configmap `trusted-ca-bundle` in namespace `openshift-config-managed` when `additionalTrustBundle` is specified in install-config. The installer will create the proxy `cluster` and configmap `user-ca-bundle` resources. `trusted-ca-bundle` configmap is generated by ClusterNetwork Operator at run time by validating configmap `user-ca-bundle`, merging the contents with the RHCOS trust bundle and producing configmap `trusted-ca-bundle`. Take a look at the slide for additional detail.